todo-app/src/components/CreateTodoList.js

```diff
@@ -29,8 +35,26 @@ const CreateTodoList = () => {
       )}
       {isNameList && (
         <div>
-          <h1>{nameList}</h1>
-          <button onClick={editNameListHandler}>Edit</button>
+          <div>
+            <h1>{nameList}</h1>
+            <button onClick={editNameListHandler}>Edit</button>
+          </div>
+          <form onSubmit={addTodoHandler}>
+            <input
+              placeholder="Add your todo"
+              ref={todoRef}
+              type="text"
+            ></input>
+            <button>Add</button>
+          </form>
+          {listTodo.map((item) => {
+            return (
+              <div>
```

Comment:
you could use Fragment here maybe, or at least you need to add the key prop
read more: https://reactjs.org/docs/lists-and-keys.html